recipes/photutils/meta.yaml

```diff
@@ -20,12 +20,13 @@ requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy 1.9.*  # [not py36]
```

Comment:
> Is 1.9 available on python 3.6? Right

And I am working on np17 too, but that may take a while (and very few projects will use it).